resources/lang/de.json

```
@@ -404,6 +404,7 @@
     "stationboard.arr": "an",
     "stationboard.btn-checkin": "Einchecken!",
     "stationboard.check-business": "Geschäftliche Reise",
+    "stationboard.check-chainPost": "An letzten geposteten Checkin anhängen",
```

sounds better for me. ok?

```suggestion
    "stationboard.check-chainPost": "An den letzten geposteten Checkin anhängen",
```